tests/test_models_unet.py

```diff
@@ -191,7 +191,9 @@ def dummy_input(self, sizes=(32, 32)):
         num_channels = 3
 
         noise = floats_tensor((batch_size, num_channels) + sizes).to(torch_device)
-        time_step = torch.tensor(batch_size * [10]).to(torch_device)
+        time_step = torch.tensor(batch_size * [10]).to(device=torch_device)
+        if torch_device == "mps":
+            time_step = time_step.to(dtype=torch.int32)
```

Comment:
think we can always cast it to `int32`

Reply:
The reason was to not interfere with the common path. I observed some slight differences when using `int32` instead of `int64` during the Flax tests, but they might not be relevant here.

Reply:
Both work fine with me (Flax will be in another file and surely not run on "mps")

Reply:
I meant it as a reference point, not to try to integrate it here :)